src/sysv/systemd/sssd-kcm.service.in

```diff
@@ -9,10 +9,10 @@ Also=sssd-kcm.socket
 
 [Service]
 Environment=DEBUG_LOGGER=--logger=files
-ExecStartPre=+-/bin/chown -f -R root:@SSSD_USER@ @sssdconfdir@
+ExecStartPre=+-/bin/chown -f -R -h root:@SSSD_USER@ @sssdconfdir@
 ExecStartPre=+-/bin/chmod -f -R g+r @sssdconfdir@
```

Comment:
Hi,

I wonder if it would make sense to add `--no-dereference` to `chmod` as well?

bye,
Sumit

Reply:
`chmod` doesn't have such an option (or I didn't manage to find an equivalent).

Reply:
FWIW, man page says:
```
chmod ignores symbolic links encountered during recursive directory traversals
```

Reply:
Hi,

yes, but it is delicate because man page also says "However,  for  each  symbolic  link listed on the command line, chmod changes the permissions of the pointed-to file.". Since we are only using directory names it seems safe, in my test the permissions of a file pointed to are not changed, but we have to keep in mind not to use `*` or similar after the directory name.

Btw, it looks like `chmod` from coreutils 9.x has this option.

bye,
Sumit